pastastore/connectors.py

```diff
@@ -1310,25 +1336,25 @@ def _add_item(
             item = item.to_frame()
         if isinstance(item, pd.DataFrame):
             sjson = item.to_json(orient="columns")
-            fname = os.path.join(lib, f"{name}.pas")
-            with open(fname, "w") as f:
+            fname = lib / f"{name}.pas"
+            with open(fname, "w", encoding="utf-8") as f:
```

Comment:
you can even do: `with fname.open("w", encoding="utf-8") as f` when you have an Path instance :)

Reply:
Oo, nice, I will change :+1: